components/common-go/baseserver/http.go

```diff
@@ -0,0 +1,11 @@
+package baseserver
+
+import "net/http"
+
+var healthyResponse = []byte(`healthy`)
+
+func ReadyHandler() http.Handler {
```

Comment:
registry-facade and others use a Kubernetes package for their readiness probes. We should re-use that and promote their use rather than rolling our own.

Reply:
Yep, makes sense. The idea here is that when you get a server, it already comes with sensible defaults. What does are is entirely up to us.